src/Datadog.Trace/Configuration/ConventionType.cs

```diff
@@ -0,0 +1,23 @@
+namespace Datadog.Trace.Configuration
+{
+    /// <summary>
+    /// Semantic convention used to when defining operation names, span tags, statuses etc.
+    /// </summary>
+    public enum ConventionType
+    {
+        /// <summary>
+        /// The default convention. Currently it is Datadog (sic!).
+        /// </summary>
+        Default,
+
+        /// <summary>
+        /// The OpenTelemetry convention.
+        /// </summary>
+        OpenTelemetry,
+
+        /// <summary>
+        /// The Datadog convention. Current default.
+        /// </summary>
+        Datadog,
```

Comment:
Shouldn't we use something like `Datadog = Default` so they have the same value?
```suggestion
        /// <summary>
        /// The Datadog convention. Same as <see cref="Default"/>.
        /// </summary>
        Datadog = Default
```